appsee-keep.pro

```diff
@@ -0,0 +1,7 @@
+-keep class com.appsee.** { *; }
+-dontwarn com.appsee.**
+-keep class android.support.** { *; }
+-keep interface android.support.** { *; }
+-keep class androidx.** { *; }
+-keep interface androidx.** { *; }
+-keepattributes SourceFile,LineNumberTable
```

Comment:
these proguard rules need to really be scoped down, as they stand it will keep a lot of excess methods and functionality we don't use.